src/index.js

```diff
@@ -102,8 +102,17 @@ function start () {
     logError('Missing SSO Token!')
   }
 
-  // start cron-jobs
-  store.dispatch(schedulerOneMinute())
+  setInterval(() => {
```

Comment:
I would prefer to avoid `setInterval()` and keep the cron-like approach via saga.
Is there any specific need for this change?